packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -1461,6 +1577,610 @@ const Conversation = WebexPlugin.extend({
     });
   },
 
+
+  /**
+   * common interface for facade of generator functions
+   * @typedef {object} IGeneratorResponse
+   * @param {boolean} done whether there is more to fetch
+   * @param {any} value the value yielded or returned by generator
+   */
+
+  /**
```

Comment:
Can you fill out the rest of the docs for this method?

Reply:
I wrote them in-line, just above the wrapper functions that this method returns. Each of the wrapper functions returns this IGeneratorResponse, but their docs say more specifically what's happening there. I'm much more familiar with documenting using TS interfaces and types... My jsdoc-umenting is a little rough